doc/tutorials/klondike/step5.md

```diff
@@ -413,6 +413,141 @@ FlameGame. The new world contains (almost) everything we need to play the game a
 re-created during each of the above actions.
 
 
+### KlondikeWorld class
+
+In Flame, a ```World``` is a type of ```Component``` that can contain other ```Components```,
```

Comment:
These should only have a single back-tick, tripple back-ticks are for multi-line code

Reply:
Sure! Let me change this